tests/test_pipeline.py

```diff
@@ -32,6 +32,7 @@ def _test(files: List, config_path: str) -> None:
         for i in range(expected_num_transforms - 1):
             expected_transform_path = path.join(expected_file_dir, f"transformation_{i}.cpp")
             actual_transform_path = path.join(actual_file_dir, f"transformation_{i}.cpp")
+            subprocess.check_call(["clang++", actual_transform_path, "-o", path.join(actual_path, f"{i}.o")])
```

Comment:
Should we use compiler which is provided with repo? How do we check that `clang++` is installed?

Reply:
Clang++ is pre-installed in github actions Ubuntu-latest build. I guess in task of test code compilation there is no difference between default clang++ and clang++ installed during build, but I will change default clang++ to clang installed manually in gorshochek repo